.gitlab-ci.yml

```diff
@@ -59,11 +61,7 @@ build_dogstatsd_static-deb_x64:
   tags: [ "runner:main", "size:large" ]
   script:
     - inv -e dogstatsd.build --static
-    - cp $SRC_PATH/$STATIC_BINARIES_DIR/dogstatsd $CI_PROJECT_DIR
-  artifacts:
-    expire_in: 2 weeks
-    paths:
-      - dogstatsd
+    - aws s3 cp --only-show-errors --region us-east-1 --sse AES256 $SRC_PATH/$STATIC_BINARIES_DIR/dogstatsd s3://dd-scratch-space-build-stable/$CI_PROJECT_NAME/$CI_PIPELINE_ID/dogstatsd
```

Comment:
I'm a bit weary of long lines like these, could you factor it in toplevel variables so we could write something like
`- $S3CP_CMD $SRC_PATH/$STATIC_BINARIES_DIR/dogstatsd $S3_SCRATCH_URL/$CI_PIPELINE_ID/dogstatsd`
instead?